Update/omot_024.txt

```diff
@@ -490,7 +490,7 @@ void main()
 //r母を失ってからは、…その対象は兄へと移った。叔父夫婦が憎いというよりも、兄が自分を守るために構ってもらえることが、嬉しかった。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "母を失ってからは、…その対象は兄へと移った。叔父夫婦が憎いというよりも、兄が自分を守るために構ってもらえることが、嬉しかった。",
-			NULL, "And after losing my mother... the target moved to my brother. Rather than hate my aunt and uncle, I was glad to have my brother doing what he could to protect me.", Line_Normal);
+			NULL, "And after losing my mother... the target moved to my brother. Rather than hate my aunt and uncle, I was glad my brother cared me to protect me.", Line_Normal);
```

Comment:
```suggestion
			NULL, "And after losing my mother... the target moved to my brother. Rather than hate my aunt and uncle, I was glad my brother cared enough to protect me.", Line_Normal);
```